modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/Page.es.js

```diff
@@ -176,10 +176,8 @@ const Page = ({
 	if (pages?.[activePage]) {
 		const visitor = new PagesVisitor([pages[activePage]]);
 
-		visitor.mapFields((field) => {
-			if (field.required) {
-				hasFieldRequired = true;
-			}
+		hasFieldRequired = visitor.findField((field) => {
+			return field.required;
 		});
```

Comment:
Good job, @hongvo2308. I'd like to suggest just a couple of tweaks.
Can you test that without the `return` statement, also `hasFieldRequired` is supposed to be a boolean, so you can convert the found field into a boolean like this:

```suggestion
		hasFieldRequired = !!visitor.findField((field) => field.required);
```
Thx